scripts/yast2

```diff
@@ -422,6 +422,11 @@ else
 	*) REDO_FILE=/var/lib/YaST2/restart_yast ;;
     esac
     snapshot_pre $module
+
+    # remove the redo file if it already exists before starting the module,
+    # avoid possible infinite loop (bsc#842910)
+    rm -f "$REDO_FILE"
+
```

Comment:
or does it take in count only when y2base create new one? in such case it make sense